packages/build-modules/src/index.ts

```diff
@@ -88,6 +92,8 @@ async function buildDependency (
       prepare: depNode.prepare,
       rawConfig: opts.rawConfig,
       rootModulesDir: opts.rootModulesDir,
+      scriptShell: opts.scriptShell,
+      shellEmulator: opts.shellEmulator,
```

Comment:
Why is `shellEmulator` getting added now? Missed in another PR?

Reply:
Kind of, I initially thought it would be better to only replace the shell for the project scripts only (not dependencies). But while adding this new option, I noticed that I wan't really consistent, so I added both options to all script executions.